local_costmap:
  global_frame: odom
  robot_base_frame: base_footprint

  update_frequency: 10.0
  publish_frequency: 10.0
  transform_tolerance: 0.1

  static_map: true
  rolling_window: true
  width: 3
  height: 3
  resolution: 0.05
 
  plugins:
    - {name: obstacle_layer, type: "masterarbeit::ObstacleLayer"}

  obstacle_layer:
    observation_sources: scan
    scan:
      sensor_frame: base_scan
      data_type: LaserScan
      enabled: true
      topic: scan
      marking: true
      clearing: true
      min_obstacle_height: 0.0
      max_obstacle_height: 2.0


